worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java

```diff
@@ -104,7 +103,7 @@ public synchronized void add(long currentTimestamp, N newNode) {
         _deque.add(lastNode);
```

Comment:
Seems all empty nodes have same timestamp? Maybe lastNode.getLeft() + intervalPerBucketInMills * index ?

Reply:
It seems the empty node has updated timestamp here. You can take a look at https://github.com/apache/incubator-celeborn/blob/ced12a18cd6e5c9500a25615c5e82e05bebde49f/worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java#L102

Reply:
OK